projects/plugins/boost/app/assets/src/js/features/premium-tooltip/premium-tooltip.tsx

```diff
@@ -42,10 +42,9 @@ const PremiumTooltip = () => {
 			</ul>
 
 			<div className={ styles[ 'upgrade-cta' ] }>
-				<UpgradeCTA
+				<InterstitialModalCTA
 					identifier="critical-css-tooltip"
 					description={ __( 'Automatic Critical CSS regeneration', 'jetpack-boost' ) }
-					eventName="upsell_cta_from_settings_page_tooltip_in_plugin"
```

Comment:
I've intentionally skipped the event name as the CTA will take care of it. The identifier will be useful for segmenting the exact CTA:

https://github.com/Automattic/jetpack/blob/71a3e780f64937fa8ccac3bb3243928a8e4c423b/projects/plugins/boost/app/assets/src/js/features/upgrade-cta/upgrade-cta.tsx#L27-L28